<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>288</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Windows Says</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid rgb(150, 0 , 0);
	background-color: rgb(245, 0, 0);
	border-radius: 10px;
	color: rgb(245, 0, 0);
	
}

QPushButton::pressed {
	background-color: rgb(150, 0 , 0);
	color: rgb(150, 0 , 0);
}</string>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid rgb(150, 150, 0);
	background-color: rgb(245, 245, 0);
	border-radius: 10px;
	color: rgb(245, 245, 0);
}

QPushButton::pressed {
	background-color: rgb(150, 150 , 0);
	color: rgb(150, 150 , 0);
}</string>
    </property>
    <property name="text">
     <string>yellow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid rgb(0,  0, 150);
	background-color: rgb(0, 0, 245);
	border-radius: 10px;
	color: rgb(0, 0, 245);
}

QPushButton::pressed {
	background-color: rgb(0, 0 , 150);
	color: rgb(0, 0 , 150);
}</string>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="greenButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 2px solid rgb(0, 150 , 0);
	background-color: rgb(0, 245, 0);
	border-radius: 10px;
	color: rgb(0, 245, 0);
}

QPushButton::pressed {
	background-color: rgb(0, 150 , 0);
	color: rgb(0, 150 , 0);
}</string>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(220, 220, 220);
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string> Hi-Score:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="highscoreLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>  Developed for Microsoft® Windows® XP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>definitely</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yellowButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startClicked()</slot>
  <slot>colorClicked()</slot>
 </slots>
</ui>
